### ISSUE TITLE
Provision Woocommerce webhooks through its API

### ISSUE THREAD

**Issue by sauloperez** [coopdevs/lazona_connector#5]
This aims to guarantee that the Woocommerce site we are listening to has the necessary webhooks properly configured for this integration to work. This should also be idempotent so we don't end up with duplicate incoming requests.

**Comment by sauloperez** [comment]
This is the one I had to set up in production for the shipment integration to work

![Screenshot from 2021-05-17 16-31-40](https://user-images.githubusercontent.com/762088/118506548-b3082980-b72d-11eb-9d79-57487f8251f7.png)

I'd like it to provision it by simply executing a script ala Config as Code.

**Comment by daniquilez** [comment]
Only approach I've found is creating a script using woocommerce api:
https://woocommerce.github.io/woocommerce-rest-api-docs/#webhooks

**Comment by sauloperez** [comment]
Exactly. That was the idea. I would implement it as a custom management command as explained on https://docs.djangoproject.com/en/3.2/howto/custom-management-commands/.
